pytest_logger/plugin.py

```diff
@@ -97,9 +99,20 @@ def pytest_runtest_teardown(self, item, nextitem):
         if logger:
             logger.on_teardown()
 
+    @pytest.mark.hookwrapper
     def pytest_runtest_makereport(self, item, call):
+        outcome = yield
+        tr = outcome.get_result()
         logger = getattr(item, '_logger', None)
         if logger:
+            if not win32 and sys.version_info >= (3, 6, ) and tr.outcome == 'failed' and self._logsdir:
```

Comment:
In python3 `os.symlink` is implemented so why win32 is blacklisted here? If we removed the python version constraint `os.symlink` on windows is already handled in pytest_logger/plugin.py::_refresh_link in different way - please keep consistency.

Reply:
ok, fixed.